hack/charts/namespaced/crd_chart/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+name: amazon-sagemaker-operator-for-k8s
```

Comment:
nit: Chart name doesn't specify that it's only CRDs

Reply:
Fixed